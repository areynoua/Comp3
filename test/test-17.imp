begin
c := 3;
function @foo() do
c := 4
end;
print(c);
@foo();
print(c);
end
