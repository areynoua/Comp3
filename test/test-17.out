3
3
success
